packages/zwave-js/src/lib/controller/Controller.ts

```diff
@@ -1323,6 +1377,9 @@ export class ZWaveController
 			nodeIds.unshift(this._ownNodeId!);
 		}
 
+		// BUGBUG: do nodes need to implicitly know that they were a long range node? Or are long range nodes determined 100% by their nodeID values being >= 256?
+		// The controller is the odd-man out, as it's both. Let's assume that apart from explicit exclusion we don't need to know for now.
+		nodeIds.push(...lrNodeIds);
```

Comment:
I couldn't find a spot in the serial API that really cared "this is a long range node" or not.

We _probably_ don't want people assuming that from the nodeID though, so there's probably a boolean parameter `isLongRange` or something that we need to instantiate on the `Node` objects?